scenarios/openshift/post_action_regex.py

```diff
@@ -1,71 +1,90 @@
 #!/usr/bin/env python3
-import subprocess
+import logging
 import re
+import subprocess
 import sys
+
 from kubernetes import client, config
 from kubernetes.client.rest import ApiException
-import logging
 
 
-# List all namespaces
 def list_namespaces():
-    namespaces = []
+    """
+    List all namespaces
+    """
+    spaces_list = []
     try:
         config.load_kube_config()
         cli = client.CoreV1Api()
         ret = cli.list_namespace(pretty=True)
     except ApiException as e:
         logging.error(
-            "Exception when calling \
-                       CoreV1Api->list_namespaced_pod: %s\n"
-            % e
+            "Exception when calling CoreV1Api->list_namespaced_pod: %s\n",
```

Comment:
This PR doesn't introduce this change but can we s/list_namespaced_pod/list_namespace please?